### DISCUSSION TITLE
The GitHub Universe ‘23 agenda has landed 🚀

### DISCUSSION THREAD

**Discussion by tuves** [community/community#68412] [Announcements]
![1200x630-Blog-Master-Half@2x](https://github.com/community/community/assets/49915749/88a34264-1148-4980-8ef8-3bee513eeaca)

The GitHub Universe '23 agenda is here and more dynamic than ever! This year, we’re honing in on empowering developers, enterprise leaders, and security professionals to stay on the cutting edge of AI-powered software development. 

You can still register, and if you join us in person at the Yerba Buena Center for the Arts in San Francisco before October 6, you'll snag our Early Bird discount. Virtual attendance is free for those who can't make it in person.

**[Get them here!](https://githubuniverse.com/?utm_source=Discussion&utm_medium=GitHub&utm_campaign=schedule-launch)**
Build today, transform tomorrow at Universe ‘23 💪
We've carefully curated two days filled with AI, security, and DevEx sessions to match your learning style. Whether you prefer insights from industry experts in our general sessions, hands-on learning at the Demo Stage, or in-depth discussions in the Discussions Lounge, we've got you covered. Take a look at some of our highlighted sessions below! 

**[Collaborating effectively using GitHub](https://reg.githubuniverse.com/flow/github/universe23/sessioncatalog/page/sessioncatalog?search=DEV1210G)**
Rob Bos, Xpirit | Xebia
In-person and virtual

**[Unlocking the power of AI in open source the ethical way](https://reg.githubuniverse.com/flow/github/universe23/sessioncatalog/page/sessioncatalog?search=AI2658L)**
Quiana Berry, RedHat
In-person only

**[Enhancing code security with the power of GitHub Copilot](https://reg.githubuniverse.com/flow/github/universe23/sessioncatalog/page/sessioncatalog?search=SEC1068G)**
Joseph Katsioloudes, GitHub
In-person and virtual

**[How to do what AI just CAN'T: Proving your value when code is commoditized](https://reg.githubuniverse.com/flow/github/universe23/sessioncatalog/page/sessioncatalog?search=AI2400L)**
Brook Riggio, Code Fellows
In-person only

**[Revolutionizing development: Mercedes-Benz's journey with GitHub Copilot](https://reg.githubuniverse.com/flow/github/universe23/sessioncatalog/page/sessioncatalog?search=AI2279G)**
Jasmine Gerth-Ramos, Holger Fahner, Mercedes Benz
In-person and virtual

Don't forget to explore the [full agenda](https://reg.githubuniverse.com/flow/github/universe23/sessioncatalog/page/sessioncatalog) and customize your learning journey using our schedule builder once you've registered.

https://github.com/community/community/assets/49915749/64814dec-0f9f-4178-bfd1-0b55d03630c4

**Git GitHub certified or sign up for a workshop 📚** 
And that's not all! Our in-person Universe '23 experience offers additional learning and networking opportunities, including workshops from November 7-9. If you attend these workshops, you'll even receive a free voucher to become GitHub certified in Foundations, Actions, Security, or Administration. It's a valuable addition to your resume that showcases your proficiency with GitHub’s platform. 

Hurry, space is limited! 
So, what are you waiting for?

[Grab your tickets](https://githubuniverse.com/?utm_source=Discussion&utm_medium=GitHub&utm_campaign=schedule-launch) for GitHub Universe '23 and prepare to dive into the future of AI-powered software development, connect with peers, and elevate your career.

Take a look at the [full course catalog](https://reg.githubuniverse.com/flow/github/universe23/sessioncatalog/page/sessioncatalog). Which sessions are your favorites? Let us know in the comments below. 👇

**Comment by Promisekevin** [comment]
🌹🌍

**Comment by 2Timmy11** [comment]
Ok I will

**Comment by 2Timmy11** [comment]
Ok, let me see what I can do.

**Comment by ghost** [comment]
Github's forced 2FA requirement for people not even in an organization and only doing casual projects is absolute garbage. Screw your agenda.

**Reply by Duckerducker** [reply]
How about the KKK oz... lol 🖕
People like you is the reason why you name yourself ghost and why GitHub is shit.
I hope you die the next time you get into a car.  No go eat dick🖕

**Reply by S-Oktay-Bicici** [reply]
I agree

**Reply by Carlojay17** [reply]
Ok

**Comment by Pikamon46** [comment]
👍What next?

**Comment by muteb9000000** [comment]
muteb

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
